src/workerd/io/worker-interface.capnp

```diff
@@ -193,6 +207,15 @@ interface EventDispatcher @0xf20697475ec1752d {
   # the success of the batch, including which messages should be considered acknowledged and which
   # should be retried.
 
+  jsRpcService @9 () -> (server :JsRpcTarget);
```

Comment:
For consistency with `getHttpService()`:

```suggestion
  getJsRpcService @9 () -> (server :JsRpcTarget);
```

Reply:
Or maybe `getJsRpcTarget`? (bikeshedding...)

Reply:
https://github.com/cloudflare/workerd/pull/1028/commits/f40a204584e02c9886b5265b2a7a104f7a1e225d